From 3d36923eb54b98ab5b29ff09d37982a642bf4b2d Mon Sep 17 00:00:00 2001
From: Maksym Sditanov <feniksa@rambler.ru>
Date: Thu, 7 Feb 2013 16:38:49 +0200
Subject: [PATCH] Surround with quotes variables like CC

When CC variable contain "gcc -m32" build fails. So quotes will fix
this bug.
---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 42c4952..6e65ed9 100644
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@ OPT ?= -O2 -DNDEBUG       # (A) Production use (optimized mode)
 #-----------------------------------------------
 
 # detect what platform we're building on
-$(shell CC=$(CC) CXX=$(CXX) TARGET_OS=$(TARGET_OS) \
+$(shell CC="$(CC)" CXX="$(CXX)" TARGET_OS="$(TARGET_OS)" \
     ./build_detect_platform build_config.mk ./)
 # this file is generated by the previous line to set build flags and sources
 include build_config.mk
-- 
1.7.9.5

